src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/TaskbarJumpList.cs

```diff
@@ -16,6 +16,7 @@ internal static void CreateElevatedEntry(string title)
             // Check startupInfo first to know if the current shell is interactive and owns a window before proceeding
```

Comment:
@bergmeister, your last commit had 152 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-ExperimentalFeature will disable Experimental Feature for scope: AllUsers
```powershell
Expected $true, because Test config should enable ExpTest.FeatureOne, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 81
81:         $feature.Enabled | Should -BeTrue -Because "Test config should enable ExpTest.FeatureOne"
```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Disable-ExperimentalFeature will disable Experimental Feature for scope: CurrentUser
```powershell
Expected $true, because Test config should enable ExpTest.FeatureOne, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 81
81:         $feature.Enabled | Should -BeTrue -Because "Test config should enable ExpTest.FeatureOne"
```
Testing Get-Help Progress.Last ProgressRecord should be Completed
```powershell
Expected Completed, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.Tests.ps1: line 106
106:             $j.ChildJobs[0].Progress[-1].RecordType | Should -Be ([System.Management.Automation.ProgressRecordType]::Completed)
```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Enable-ExperimentalFeature will enable Experimental Feature for scope: CurrentUser
```powershell
Expected $null or empty, because No object is output to pipeline on success, but got GetStartupInfo.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\EnableDisable-ExperimentalFeature.Tests.ps1: line 64
64:         $feature | Should -BeNullOrEmpty -Because "No object is output to pipeline on success"
```
Get-ExperimentalFeature Tests.Feature disabled tests.'Get-ExperimentalFeature' should return all available features from module path
```powershell
Expected exactly 'ExpTest.FeatureOne', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 60
60:             $features[0].Name | Should -BeExactly "ExpTest.FeatureOne"
```